train_configs/debug_model.toml

```diff
@@ -35,7 +35,7 @@ warmup_steps = 2  # lr scheduler warm up, normally 20% of the train steps
 max_norm = 1.0  # grad norm clipping
 steps = 10
 data_parallel_degree = -1
-tensor_parallel_degree = 1
+tensor_parallel_degree = 2
```

Comment:
do we want to leave tp on as default?
maybe have a debug_model_tp.toml

Reply:
cc @wanchaol @tianyu-l

Reply:
no I don't think we should make it as default. Also we should not have a separate toml file, we should simply add a integration test entry in test_runner.py